cloud/credentials.go

```diff
@@ -420,6 +420,12 @@ type CredentialAttr struct {
 
 	// Options, if set, define the allowed values for this field.
 	Options []interface{}
+
+	// ShortSuffix is a human-readable suffix that we add to the name of
+	// the attribute when prompting. This is only used in the case of last
+	// interactive attributes, and it is useful to explain the user what
+	// to insert.
```

Comment:
```suggestion
	// ShortSuffix is a human-readable suffix that we add to the name of
	// the attribute when prompting. This replaces the (optional) suffix
	// to prompt users about why this is optional. (requires setting Optional: true)
```

I don't think this wording is perfect, but I would mention that ShortSuffix doesn't do anything without Optional, and that it replaces the (optional) suffix with this suffix.

Reply:
Yeah good catch. Updated it.